src/pages/Docchi/meta.json

```diff
@@ -0,0 +1,5 @@
+{
+  "urls": {
+    "match": ["*://docchi.pl/anime/*"]
```

Comment:
Because it is a single page app it needs to inject on all pages. `*://docchi.pl/*`

Reply:
Updated